tensorflow/python/ops/math_ops.py

```diff
@@ -2786,7 +2786,17 @@ def add_n(inputs, name=None):
   """Adds all input tensors element-wise.
 
   Converts `IndexedSlices` objects into dense tensors prior to adding.
-
+  tf.math.add_n performs same operation as'tf.math.accumulate_n' but it waits
```

Comment:
`tf.math.add_n` should have \` around it (e.g. `` `tf.math.add_n` ``). If you do that our doc generator will correctly add links between the pages.

Please also replace the single quotes around `'tf.math.accumulate_n'` with \` too.

Finally, grammar: "performs **the** same" (add "the").